styleguide/_themes/global/scss/fonts.scss

```diff
@@ -2,11 +2,13 @@ $heading: 'Khand', Helvetica, Arial, sans-serif;
 $subhead: 'Khand', Helvetica, Arial, sans-serif;
 $copy: 'Open Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
 $code: Tahoma, Arial, Geneva, Helvetica;
+$font-rsweb: 'rsweb';
```

Comment:
@brantdarilek Good call. I was originally going to attempt to remove the need for the `.rsweb` overwrite but the specificity was causing problems. I meant to remove but didn't. I have updated the PR. Let me know if you see any other edits. Thanks!